recipe/build.sh

```diff
@@ -37,6 +37,7 @@ fi
             --enable-unsupported \
             --enable-using-memchecker \
             --enable-static=yes \
+            --enable-ros3-vfd \
```

Comment:
last element shouldn't have a trailing slash

Reply:
I thought it was odd but the static one had it too. Any idea why that works?

Reply:
Probably because there are just blank lines following it